src/button/button-group.scss

```diff
@@ -5,72 +5,79 @@
 @mixin iui-button-group {
   display: inline-flex;
   align-items: center;
+  isolation: isolate;
```

Comment:
Any alternatives to support IE?

Reply:
"Supporting IE" does not mean everything has to work perfectly. This property is just to seal off the z-index. If the z-index is leaking in IE, I don't think it's a big deal as it's just on hover/focus.